File: Documentation/3.5/Samples/csharp/Raven.Documentation.Samples/FileSystem/ClientApi/Session/Configuration/MaxRequests.cs
namespace Raven.Documentation.Samples.FileSystem.ClientApi.Session.Configuration
{
	using Client.FileSystem;

	public class MaxRequests
	{
		public MaxRequests()
		{
			using (var store = new FilesStore())
			{
				using (var session = store.OpenAsyncSession())
				{
					#region max_requests_1
					session.Advanced.MaxNumberOfRequestsPerSession = 50;
					#endregion
				}

				#region max_requests_2
				store.Conventions.MaxNumberOfRequestsPerSession = 100;
				#endregion
			}
		} 
	}
}

File: Documentation/5.4/Samples/java/src/test/java/net/ravendb/ClientApi/Session/Configuration/MaxRequests.java
package net.ravendb.ClientApi.Session.Configuration;

import net.ravendb.client.documents.DocumentStore;
import net.ravendb.client.documents.IDocumentStore;
import net.ravendb.client.documents.session.IDocumentSession;

public class MaxRequests {
    public MaxRequests() {

        try (IDocumentStore store = new DocumentStore()) {
            try (IDocumentSession session = store.openSession()) {
                //region max_requests_1
                session.advanced().setMaxNumberOfRequestsPerSession(50);
                //endregion
            }

            //region max_requests_2
            store.getConventions().setMaxNumberOfRequestsPerSession(100);
            //endregion
        }
    }
}
